nixos/modules/system/boot/systemd/oomd.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, ... }: let
+
+  cfg = config.systemd.oomd;
+
+in {
+  options.systemd.oomd = {
+    enable = lib.mkEnableOption "the systemd-oomd OOM killer" // { default = true; };
+
+    enableRootSlice = lib.mkEnableOption "oomd on the root slice (-.slice)";
+    enableSystemSlice = lib.mkEnableOption "oomd on the system slice (system.slice)";
+    enableUserServices = lib.mkEnableOption "oomd on all user services (user@.service)";
```

Comment:
What's the defaults for Fedora on these? Is this desirable to have enabled by default?

Reply:
The the comment by @QuantMint. I'd enable it for the system slice on my servers but for now I think it's better for the killer to be opt-in.

Fedora's defaults are `enableRootSlice = true` and `enableUserServices = true`

Reply:
I added a comment on what Fedora does